bitnami/kubeapps/README.md

```diff
@@ -51,7 +52,417 @@ Once you have installed Kubeapps follow the [Getting Started Guide](https://gith
 
 ## Parameters
 
-For a full list of configuration parameters of the Kubeapps chart, see the [values.yaml](values.yaml) file.
+### Global parameters
+
+| Name                      | Description                                     | Value |
+| ------------------------- | ----------------------------------------------- | ----- |
+| `global.imageRegistry`    | Global Docker image registry                    | `nil` |
+| `global.imagePullSecrets` | Global Docker registry secret names as an array | `[]`  |
+| `global.storageClass`     | Global StorageClass for Persistent Volume(s)    | `nil` |
+
+
+### Common parameters
+
+| Name                      | Description                                                                   | Value   |
+| ------------------------- | ----------------------------------------------------------------------------- | ------- |
+| `kubeVersion`             | Override Kubernetes version                                                   | `nil`   |
+| `nameOverride`            | String to partially override common.names.fullname                            | `nil`   |
+| `fullnameOverride`        | String to fully override common.names.fullname                                | `nil`   |
+| `commonLabels`            | Labels to add to all deployed objects                                         | `{}`    |
+| `commonAnnotations`       | Annotations to add to all deployed objects                                    | `{}`    |
+| `extraDeploy`             | Array of extra objects to deploy with the release                             | `[]`    |
+| `allowNamespaceDiscovery` | Allow users to discover available namespaces (only the ones they have access) | `true`  |
```

Comment:
Looks like "common parameters" is a collection of all the top-level options in the values.yaml that aren't for a specific service? Nice. Some of them might not fit perfectly, but it's not an issue, imo. EDIT: we could move `allowNamespaceDiscovery`, `clusters` and `featureFlags` to "Other parameters"? Don't mind either way.

Reply:
Love the suggestion. I also had the feeling these ones don't fit here but I wasn't very sure where to include them.